processing_options.go

```diff
@@ -858,6 +860,14 @@ func applyStripMetadataOption(po *processingOptions, args []string) error {
 	return nil
 }
 
+func applyAutoRotateOption(po *processingOptions, args []string) error {
+	if len(args[0]) > 0 {
+		po.AutoRotate = parseBoolOption(args[0])
+	}
```

Comment:
Please add proper validation as I did for `strip_metadata` https://github.com/imgproxy/imgproxy/commit/a1679e5e092236bb19a2dabbdf728f1e40a7f9d0